From 8f32bf8b9a1513ef1be7765cae8007776decf91c Mon Sep 17 00:00:00 2001
From: Matthew Gerlach <mgerlach@opensource.altera.com>
Date: Fri, 30 Sep 2016 14:25:10 -0700
Subject: [PATCH 014/151] FogBugz #408205: fix printf format error

commit  13da642beeaa5516641c3554c580590d681306b8 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

The version 4.8.5 of gcc complains when %d is
used to printf a variable of size_t.  Using %zu
fixes the problem.

Signed-off-by: Matthew Gerlach <matthew.gerlach@intel.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 drivers/fpga/fpga-mgr-debugfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/fpga/fpga-mgr-debugfs.c b/drivers/fpga/fpga-mgr-debugfs.c
index 55731816bf92..b25e337cb7ae 100644
--- a/drivers/fpga/fpga-mgr-debugfs.c
+++ b/drivers/fpga/fpga-mgr-debugfs.c
@@ -103,7 +103,7 @@ static ssize_t fpga_mgr_image_write_file(struct file *file,
 	char *buf;
 	int ret;
 
-	dev_info(&mgr->dev, "writing %d bytes to %s\n", count, mgr->name);
+	dev_info(&mgr->dev, "writing %zu bytes to %s\n", count, mgr->name);
 
 	buf = kzalloc(count, GFP_KERNEL);
 	if (!buf)
-- 
2.26.1

